src/utils/FormInputHandling.ts

```diff
@@ -17,14 +17,65 @@ async function getData(currentPage: number, limit: number) {
 
 // post data to the server
 async function postFormData(formData: RowType) {
-  const currentDate = new Date().toLocaleDateString();
+  const { student, test, timeline, dateCreated } = formData;
+  let start_time = timeline.startDate + "T" + timeline.startTime + ":00";
+  let end_time = timeline.endDate + "T" + timeline.endTime + ":00";
```

Comment:
same - check for end time.. 

row["start_date"] in my code looks like "2023-11-12" for example
and row["start_time"] looks like 6:00:00 pm for example

check one of the older session rows in staging-db and see if ur start time end time formats are matching with that row. this is quite important.

Reply:
staging-db returns this  "start_time": "2023-11-17T01:21:00Z", and then I am destructuring it